parity/reports/algod2indexer_dropped.yml

```diff
@@ -0,0 +1,30 @@
+ApplicationParams:
```

Comment:
@winder - are these "dropped" algod fields as expected?

Reply:
`creator` is expected.

Do the `INDEXER: null` lines mean indexer doesn't have those objects? I think these are correct in that case. The object that needs to be identical between algod/indexer is Account (and its dependencies).

Reply:
Yes, `INDEXER: null` indicates that we expect the path not to exist, so in this case not to have a `creator` on indexer.